manifests/init.pp

```diff
@@ -292,6 +292,22 @@
   if $pulpcore and !$pulpcore_mirror {
     include foreman
 
+    if $pulp {
```

Comment:
It shouldn't matter, but I would move this below the pulpcore definition. That way all plugins are defined after the class itself and keeps them a bit grouped. In the future the plugin define might include pulpcore and then the current implementation will fail to compile due to a duplicate class definition.

Reply:
@ekohl done.